unistd/sys.c

```diff
@@ -60,6 +61,23 @@ static int shebang(const char *path)
 	return fd;
 }
 
+static int checkelf(const char *path)
+{
```

Comment:
We can drop this after/if https://github.com/phoenix-rtos/phoenix-rtos-kernel/pull/269 is merged

Reply:
Mentioned commit merged. Whole `checkelf()` addition was removed from commit